xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-ui/src/main/resources/XWiki/Like/LikeUIX.xml

```diff
@@ -162,21 +173,30 @@ require(['jquery', 'xwiki-meta', 'xwiki-l10n!xwiki-like-messages'], function ($,
     }
   };
 
+  var getLikersLinkText = function(nb) {
+    // conditional parametrized translation formatting seems to not work with the javascript l10n API,
+    // so we're using this fallback method with 3 keys
+    return l10n.get('likers.link.' + (nb == 0 ? 'zero' : (nb &lt;= 1 ? 'one' : 'many')), nb);
```

Comment:
The proper policy would be https://dev.xwiki.org/xwiki/bin/view/Community/L10N/Conventions/#HRulesfortranslatingcompositeandlongmessages , allowing the translater to handle variation with the value as they need for each specific language.

It's just that it doesn't work in javascript localization.
Ideally, the like service should send this text as well (in which case we can use proper conditional translations), as I made it send the new button content, but it was too much of a refactoring for a first implementation of XWIKI-18545. It can be done in a future version.